### PR TITLE
Portfolio toolbar headline

### PR DESCRIPTION
**Pull request by ghost**
The end result I got was pretty similar in my point of view, but I had difficulty making the fonts bolder to match the ones in Figma. Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <div>
+                <a  class="logo" href="#">My Logo</a>
+            </div>
+            <div class="menu-placeholder">
+                <a href="#"><img src="./images/Union.svg" alt="menu icon"></a>
+            </div>
```
Comment (karvel-code) [inline index.html]: - [x] Please change the `<div>` tags to `<nav>`, this is so as to enhance the semantics of your HTML code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="connect">Let's connect</a>
+        </div>
+        <div>
+            <ul class="social">
+                <li class="social-icon"><a href="#" class="twitter"><img src="./images/twitter.png" alt="twitter icon"></a></li>
+                <li class="social-icon"><a href="#" class="linkedin"><img src="./images/linkedin.png" alt="linkedin icon"></a></li>
+                <li class="social-icon"><a href="#" class="medium"><img src="./images/medium.png" alt="medium icon"></a></li>
+                <li class="social-icon"><a href="#" class="github"><img src="./images/github.png" alt="github icon"></a></li>
+                <li class="social-icon"><a href="#" class="angellist"><img src="./images/angellist.png" alt="angellist icon"></a></li>
```
Comment (karvel-code) [inline index.html]: - [x] For the main body section, please enclose the HTML code using the `<section>` tag. This is in order to enhance the semantics of your HTML code.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @iKuartz , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight
- You have highly organized code.
- All your linter checks are passing successfully.
- You have adhered to git-flow.

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

```

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: Yes, makes sense since it will be a navigation menu. Thank you.

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html]: Ok, done! Thank you!

**Review by HENRYKC24 — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: 
Approved
## Hi @iKuartz :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**